configs/nvme-storage-configurator/base/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: nvme-storage-config
+resources:
+  - serviceaccount.yaml
+  - clusterrolebinding.yaml
+  - daemonset.yaml
+configMapGenerator:
+- name: nvme-storage-configurator
```

Comment:
you need to mention the namespace that will be used for the reource in here. In addition you need to create a resource for the namespace itself.

Reply:
added namespae resource